config/ember-try.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-env node */
```

Comment:
At some point I was thinking that it would be nice to switch to using ember-try for the tests, but until then I'll remove it.